src/App.tsx

```diff
@@ -1,14 +1,55 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
-import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
-import { TodoModal } from './components/TodoModal';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 import { Loader } from './components/Loader';
+import { TodoList } from './components/TodoList';
+import { TodoModal } from './components/TodoModal';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+
+  const [selectTodo, setSelectTodo] = useState(0);
+  const [isLoad, setIsLoad] = useState(true);
+
+  useEffect(() => {
+    const loadTodos = async () => {
+      const listTodos = await getTodos();
+
+      setTodos(listTodos);
+      setIsLoad(false);
+      setVisibleTodos(listTodos);
+    };
+
+    loadTodos();
+  }, []);
+
+  const todosFilter = (inputType: string, typeSelect: string) => {
+    let visibleMovies = todos.filter(
+      ({ title }) => title.toLowerCase().includes(inputType),
+    );
+
+    if (typeSelect === 'active') {
+      visibleMovies = todos
+        .filter(({ completed }) => completed === false);
+    }
+
+    if (typeSelect === 'completed') {
+      visibleMovies = todos
+        .filter(({ completed }) => completed === true);
+    }
```

Comment:
You can use `filter` once, and inside the filter callback check these conditions